modules/guides/pages/transactions.adoc

```diff
@@ -0,0 +1,603 @@
+= Couchbase Transactions
+:page-topic-type: guide
+:imagesdir: ../assets/images
+:tabs:
+:description: How to create Couchbase transactions using N1QL.
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+[abstract]
+{description}
+
+== Introduction
+
+Couchbase transactions enable you to carry out ACID (atomic, consistent, isolated, and durable) actions on the database.
+This how-to guide covers N1QL support for Couchbase transactions.
+Some SDKs also support Couchbase transactions.
+Refer to <<related-links>> for further details.
+
+Only DML (data modification language) statements are permitted within a transaction:
+xref:n1ql:n1ql-language-reference/insert.adoc[INSERT],
+xref:n1ql:n1ql-language-reference/upsert.adoc[UPSERT],
+xref:n1ql:n1ql-language-reference/delete.adoc[DELETE],
+xref:n1ql:n1ql-language-reference/update.adoc[UPDATE],
+xref:n1ql:n1ql-language-reference/merge.adoc[MERGE],
+xref:n1ql:n1ql-language-reference/select.adoc[SELECT],
+xref:n1ql:n1ql-language-reference/execfunction.adoc[EXECUTE FUNCTION],
+xref:n1ql:n1ql-language-reference/prepare.adoc[PREPARE], or
+xref:n1ql:n1ql-language-reference/execute.adoc[EXECUTE].
+
+include::partial$before-you-begin.adoc[tag=body]
+include::partial$query-tools.adoc[tags=body;!thumbs]
+
+include::partial$data-warning.adoc[]
```

Comment:
ooo, this partial is useful "Please note that the examples in this guide will alter the data in your sample buckets."
@maria-robobug @RichardSmedley we could do with this in our SDK examples that work against travel-sample too?

Reply:
Yeah, could be added in SDK common or something. Guess we would have to include it in every SDK page with examples that alter data.